include/linux/resctrl.h

```diff
@@ -6,6 +6,10 @@
 #include <linux/list.h>
 #include <linux/pid.h>
 
+/* CLOSID, RMID value used by the default control group */
+#define RESCTRL_RESERVED_CLOSID		0
```

Comment:
These lines are not part of this commit in upstream https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=6791e0ea30711b937d5cb6e2b17f59a2a2af5386 but they are part of this commit Track the closid with the rmid https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=40fc735b78f0c81cea7d1c511cfd83892cb4d679

Reply:
@clsotog you could compare with MPAM-dev[0]
the lines are added in next commit x86/resctrl: Access per-rmid structures by index.
It was a clean cherry-picked and built amd64|arm64 successfully so I didn't find the difference.
Do you want me to remove these lines in this commit?
Thanks
[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev

Reply:
Im sure it will build but do we have to match upstream code?

Reply:
Okay, so let me clean up first, thanks

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev

Reply:
Have you looked at the[ ](https://git.kernel.org/pub/scm/linux/kernel/git/morse/linux.git/log/?h=mpam/snapshot/v6.7-rc2)? The patches seems a little closer to the patches that have been accepted upstream than [0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev

Reply:
@clsotog thanks,
I have worked with Tushar and 
1. backported most patches from JM_6.7-rc2
2. backported nvidia workaround from JM_6.10-rc1
3. backported those don't land in upstream&mainline kernel but are mandatory for MPAM.

Reply:
Im almost done with the JM_6.7-rc2 ones but one question the one backported that dont land in upstream are like this one 413a7709760c fs/resctrl: Add this_is_not_abi mount option. Is this one like needed for testing?

Reply:
AFAIK its is needed to bind resctrl fs for kernels >=6.2

Reply:
@clsotog I just keep the mpam usage as same as shared.
So this commit is essential.